models/Course.js

```
@@ -0,0 +1,39 @@
+    }],
+    educator: { type: String, ref: 'User', required: true },
+    enrolledStudents: [
+        { type: String, ref: 'User' }
+    ]
+}, { timestamps: true, minimize: false });
+
+const ChapterSchema = Schema({
+    chapterId: { type: String, required: true },
+    chapterOrder: { type: Number, required: true },
+    chapterTitle: { type: String, required: true },
+    chapterContent: [lectureSchema],
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)